tools/depends/native/TexturePacker/src/TexturePacker.cpp

```diff
@@ -92,11 +92,17 @@ void Usage()
 {
   puts("Usage:");
   puts("  -help            Show this screen.");
+  puts("  -version         Show version.");
   puts("  -input <dir>     Input directory. Default: current dir");
   puts("  -output <dir>    Output directory/filename. Default: Textures.xbt");
   puts("  -dupecheck       Enable duplicate file detection. Reduces output file size. Default: off");
 }
 
+void Version()
+{
+  puts("TexturePacker version 1.0.0");
```

Comment:
Magic version string?

Reply:
No. Texturepacker should be standalone. It should handle it's versioning itself. The VERSION file should then match that, not the other way around

Reply:
Got it, VERSION file should match the version here. Still a magic string (non-code value that is mixed in with code).

Reply:
Open to improvements, I just don't really want to spend time on it, hence the quick and dirty

Reply:
You could create a `Version.h` file that only defines 1.0.0, then include it and use it in the `--version` flag. Better to not include the use of magic strings in the codebase.